src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -123,22 +144,16 @@ export function Tokens() {
     },
   );
 
-  const allAssets = useMemo(
-    () =>
-      uniqBy(
-        [...assets, ...customNetworkAssets].sort(
-          (a: ParsedUserAsset, b: ParsedUserAsset) =>
-            parseFloat(b?.native?.balance?.amount) -
-            parseFloat(a?.native?.balance?.amount),
-        ),
-        'uniqueId',
-      ),
+  const combinedAssets = useMemo(
```

Comment:
@KosmosKey Looks like there is a merge conflict with #1383 here and that new logic should be kept. I think we also typically perform filters in-line instead of a separate hook, so I'd suggest we refactor to shift to that. Maybe a `usePinnedAssets` hook would be broken out, and then the combined filters would be inline here